protobufs/livekit_analytics.proto

```diff
@@ -110,8 +111,17 @@ enum AnalyticsEventType {
   INGRESS_STARTED = 16;
   INGRESS_ENDED = 17;
   INGRESS_UPDATED = 29;
-
-  // NEXT_ID: 30
+  SIP_INBOUND_TRUNK_CREATED = 30;
+  SIP_INBOUND_TRUNK_DELETED = 31;
+  SIP_OUTBOUND_TRUNK_CREATED = 32;
+  SIP_OUTBOUND_TRUNK_DELETED = 33;
+  SIP_DISPATCH_RULE_CREATED = 34;
+  SIP_DISPATCH_RULE_DELETED = 35;
+  SIP_PARTICIPANT_CREATED = 36;
```

Comment:
`SIP_PARTICIPANT_CREATED` is basically `SIP_CALL_STARTED` + outbound flag. If we are adding this one, we probably should add `SIP_CALL_ACCEPTED` to reflect the same event for inbound.